home-screen-widget/profiles/README.md

```diff
@@ -0,0 +1,23 @@
+#Home Screen Example Widget
+
+This is a guided example on how to develop a home screen widget.
+In this example we provide a working widget that uses the Beekeeper Javascript SDK to fetch 50 user profiles and display
+them in a grid layout.
+
+For a more in depth overview on how the home screen is built and on how to set up the development environment
+please visit our [developer portal](https://developers.beekeeper.io/v2/welcome/home-screen)  
+
+## Limitations
+
+* The home screen only supports widgets developed using the Vue.js framework. To find out more about Vue.js click [here](https://vuejs.org/v2/guide/)
```

Comment:
Can you add Vuex as another dependency? Also, can you add major version numbers for both dependencies?